app.yaml

```diff
@@ -14,79 +14,54 @@ inbound_services:
 - warmup
 - mail
 
+default_expiration: "10m"
 handlers:
 - url: /favicon.ico
   static_files: assets/favicon.ico
   upload: assets/favicon.ico
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /robots.txt
   static_files: assets/robots.txt
   upload: assets/robots.txt
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /sitemap.xml
   static_files: assets/sitemap.xml
   upload: assets/sitemap.xml
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /build
   static_dir: build
   secure: always
-  expiration: 30d
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
+  expiration: "90d"
 - url: /assets
   static_dir: assets
   secure: always
   application_readable: true
-  http_headers:
-    Cache-Control: 'public, max-age=60'
 # Serve js scripts and css files under core/templates/dev/head.
 # This regex allows us to recursively serve js scripts.
 # "\1" inserts text captured by the capture group in the URL pattern.
 - url: /templates/dev/head/(.*\.(js|css))$
   static_files: core/templates/dev/head/\1
   upload: core/templates/dev/head/(.*\.(js|css))$
   secure: always
-- url: /templates/dev/head/(.*\.(html))$
+- url: /templates/dev/head/(.*directive\.html)$
```

Comment:
Could you explain this change (and the one just below)?

Reply:
This handler was added to enable usage of `templateUrl` in directives. Now that all directives use `templateUrl` we don't need `application_readable: true` for directives, but we need it for classic html files that we serve. So I changed this handler to only serve directives HTML files so that nondirective HTML files can be loaded and served by our backend.

Reply:
Ah, OK. SGTM. Thanks for explaining!